### PR TITLE
Very small tweaks to bsfUtility/Allocators/

### PR DESCRIPTION
**Pull request by jonesmz**

### CODE REVIEW

File: Source/Foundation/bsfUtility/Allocators/BsFrameAlloc.cpp [hunk 1]
```diff
@@ -19,18 +19,10 @@ namespace bs
 		mFreePtr = 0;
 	}
 
-#if BS_DEBUG_MODE
```
Comment (jonesmz) [inline Source/Foundation/bsfUtility/Allocators/BsFrameAlloc.cpp]: The two versions of the constructor are identical, aside from formatting.

File: Source/Foundation/bsfUtility/Allocators/BsFrameAlloc.cpp [hunk 2]
```diff
@@ -120,7 +112,7 @@ namespace bs
 	{
 		// Dealloc is only used for debug and can be removed if needed. All the actual deallocation
 		// happens in clear()
-			
+
```
Comment (jonesmz) [inline Source/Foundation/bsfUtility/Allocators/BsFrameAlloc.cpp]: Superfluous whitespace

File: Source/Foundation/bsfUtility/Allocators/BsMemoryAllocator.h [hunk 3]
```diff
@@ -423,7 +423,7 @@ namespace bs
 			if (num == 0)
 				return nullptr;
 
-			if (num > std::numeric_limits<size_t>::max() / sizeof(T))
```
Comment (jonesmz) [inline Source/Foundation/bsfUtility/Allocators/BsMemoryAllocator.h]: max_size() does the same calculation.

**Comment by BearishSun** [comment]
Looks good, thank you.

**Pull request merged**